docs/user/library/coverage/geotiff.rst

```diff
@@ -5,7 +5,6 @@ The gt-geotiff module in the plugin group provides access to the GeoTiff file fo
 
 **References**
 
-* http://www.jroller.com/jsight/entry/back_to_code_geotools_geotiff
```

Comment:
I did do a quick search and couldn't find anything. Will fix if there is a valid replacement.